azure-functions/acbs-function/.dockerignore

```diff
@@ -39,13 +39,13 @@ coverage
 .vscode/tasks.json
 .vscode/launch.json
 .vscode/extensions.json
-**/report/*
+reporting/tfm/report/*
 **/generated_reports
 **/temp/*
-**/actionsheets/*
-**/json/*
-**/amendments/*
-**/exports/*
+utils/**/actionsheets/*
+utils/**/json/*
```

Comment:
Just to flag that Docker would only try to containerise the files from its current folder. I.e. the Dockerfile from acbs-functions folder wouldn't have access to utils, TFM, reporting, etc folders

Reply:
Hello, yes I agree. I have added it in case we need to any exports or reports we can use these safe paths.